pkg/model/iam/iam_builder.go

```diff
@@ -235,6 +235,54 @@ func (b *PolicyBuilder) BuildAWSPolicy() (*Policy, error) {
 	return p, nil
 }
 
+// BuildAWSPolicy generates a custom policy for a Kubernetes master.
+func (r *NodeRoleAPIServer) BuildAWSPolicy(b *PolicyBuilder) (*Policy, error) {
+	resource := createResource(b)
```

Comment:
Can we extract a common "control plane" method which both NodeRoleMaster and NodeRoleAPIServer call into (likely passing their role)?  Otherwise there's a lot of duplication here.